app/assess/data.py

```diff
@@ -140,8 +140,9 @@ def get_round(fund_id: str, round_id: str) -> Union[Round, None]:
 
 def get_bulk_accounts_dict(account_ids: List):
     account_url = Config.BULK_ACCOUNTS_ENDPOINT
-    account_params = {"account_id": list(set(account_ids))}
-    return get_data(account_url, account_params)
+    if account_ids:
```

Comment:
It might be useful to log a warn here if its called with no account ids

Reply:
Thank you for spotting. Logs added